src/neovis.js

```diff
@@ -253,16 +276,16 @@ export default class NeoVis {
 
 	// public API
 
-	render() {
+	render(query) {
 
 		// connect to Neo4j instance
 		// run query
 		let recordCount = 0;
-
-		let session = this._driver.session();
+		const _query = query ? query : this._query;
```

Comment:
I think as you should do `query || this._query` to make that more consistent with the rest of the code

Reply:
Ok will fix